src/Classes/ModStore.lua

```diff
@@ -501,8 +501,38 @@ function ModStoreClass:EvalMod(mod, cfg)
 				return
 			end
 		elseif tag.type == "SocketedIn" then
-			if not cfg or tag.slotName ~= cfg.slotName or (tag.keyword and (not cfg or not cfg.skillGem or not calcLib.gemIsType(cfg.skillGem, tag.keyword))) then
+			if not cfg or tag.slotName ~= cfg.slotName or (tag.socketColor ~= nil and (tag.sockets == nil or (tag.sockets ~= nil and type(tag.sockets) ~= "string")) and tag.socketColor ~= cfg.socketColor) or (tag.keyword and (not cfg or not cfg.skillGem or not calcLib.gemIsType(cfg.skillGem, tag.keyword))) then
 				return
+			elseif tag.sockets ~= nil then
+				if tag.sockets == "all" and tag.socketColor ~= nil then
+					local targetAtrColor = tag.socketColor == "R" and "strengthGems" or tag.socketColor == "G" and "dexterityGems" or tag.socketColor == "B" and "intelligenceGems"
+					local total = (cfg.intelligenceGems or 0) + (cfg.dexterityGems or 0) + (cfg.strengthGems or 0)
+					local count = cfg[targetAtrColor] or 0
+					if total == 0 or count == 0 or total > count then
+						return
+					end
+				elseif tag.sockets ~= nil and type(tag.sockets) == "number" and tag.socketColor ~= nil then
+					local targetAtrColor = tag.socketColor == "R" and "strengthGems" or tag.socketColor == "G" and "dexterityGems" or tag.socketColor == "B" and "intelligenceGems"
+					local count = cfg[targetAtrColor] or 0
+					if count == 0 or count < tag.sockets then
+						return
+					end
+				elseif tag.sockets ~= nil and type(tag.sockets) == "table" then
```

Comment:
```suggestion
				elseif type(tag.sockets) == "number" and tag.socketColor ~= nil then
					local targetAtrColor = tag.socketColor == "R" and "strengthGems" or tag.socketColor == "G" and "dexterityGems" or tag.socketColor == "B" and "intelligenceGems"
					local count = cfg[targetAtrColor] or 0
					if count == 0 or count < tag.sockets then
						return
					end
				elseif type(tag.sockets) == "table" then
```
No longer needed as are always true due to check it in first if statement.

Reply:
Yeah i had issues with pulling the checkout out and had to debug. Forgot to remove those after. Fixed